### DISCUSSION TITLE
Video DownloadHelper license through the Chrome Web Store

### DISCUSSION THREAD

**Discussion by vadcv** [aclap-dev/video-downloadhelper#372] [License & Purchase requests]
### What's your issue?

I purchased a Video DownloadHelper license through the Chrome Web Store (Lifetime Premium VDH) and now I can’t activate

### The 10 first digits (only 10) of your license number or email address

vadcvreg@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by Zeihold** [comment]
Same issue here:

I purchased my Chrome license on 20. February 2019 through Chrome Web Store using my email xxx@yyy.zz

And I can`t activate my license anymore.

EDIT: removed actual address because of constant attempts to trick me into activating someone elses VDH.

**Comment by marolan1** [comment]
Same problem, when i'm trying to recover license - it says that my VDH is for firefox.


removed email

**Reply by marolan1** [reply]
@jcfrog why didn't i got any response in three weeks having the same issue? I purchased lifetime license but cannot use it.

**Reply by jcfrog** [reply]
sorry I did not find any license with your email address. Can you send me proof of purchase at licenses@downloadhelper.net ?

**Comment by cristinaramoss** [comment]
Exactly the same here. purchased a Video DownloadHelper license through the Chrome Web Store (Lifetime Premium VDH) and now when i download, the premium status is not recognized asking me to purchase again. 
My email cristinaramos.studio@gmail.com

**Reply by jcfrog** [reply]
Hi, 
Sorry for that, we will send you a new license key.

**Reply by cristinaramoss** [reply]
Hi, 
may you tell me when and how shall i get it? i need to download some things and im blocked. Thanks

**Reply by cristinaramoss** [reply]
I sent you the proof of purchase to both licenses@downloadhelper.net and support@downloadhelper.net

**Comment by RobTheBob2024** [comment]
Hi I also purchased a license through the Chrome web store on 15th of April. Premium status is not recognized. My email is robbender@hotmail.com

**Reply by jcfrog** [reply]
hi,
please use this: https://www.downloadhelper.net/recover-license

**Comment by jcjjunior** [comment]
Hello
I started using it and I really like it. I wanted to buy the license key, but it's too expensive in my country. Unfortunately, our currency is not dollars or euros :/
I'm going to stay on Free and if you have any donation links to help the project, let me know. It won't be much, but it will help you.
Thank you

**Reply by mozzie4** [reply]
Is the price shown for the icence in $USD?

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.

I paid for my license in US dollars.  But that was a few years ago.  The current licensing page looks like this:

![#01](https://github.com/user-attachments/assets/1faaaabb-d4eb-409f-85ce-d719895e3273)

That's the same price I paid.  So I'd say it's still in US dollars.

**Comment by paper2Fox** [comment]
Exactly the same here. purchased a Video DownloadHelper license through the Chrome Web Store (Lifetime Premium VDH) in 2020 and now when i download, the premium status is not recognized asking me to purchase again.
My email guojiangaguojiang@gmail.com

**Comment by ephata50** [comment]
Hi I purchased a Video Downloadhelper on 30th september 2024 and I had to change of computer because the last one has crashed. My e-mail is ephata50@aol.com. Can you give me my licence key, please ?

Thanks

**Reply by jcfrog** [reply]
Hi, I sent you an email.

**Comment by ghost** [comment]
its3bdullahk7@hotmail.com
Can you give me my licence key, please ?

**Reply by jcfrog** [reply]
Hi,
Sorry I did not find your license. Please send us details at licenses@downloadhelper.net

**Comment by jiangziqiang1986** [comment]
Hello, I purchased from Chrome Web Store on October 11th, 2020. Can you give me my licence key, please? email: j***6@gmail.com

**Reply by jcfrog** [reply]
Hi, 
Sorry for that, we will send you a new license key.

**Comment by diefallen** [comment]
Hai, i bought mine via ChromeWebStore around 2020 through Googlepay, The Email was: ashe******@gmail.com
Can i get the licensekey aswell for manual activation? Thanku

EDIT: done and just censored the email (my main) incase for unwanted phisher visibility.ty

**Reply by jcfrog** [reply]
Hi, 
Sorry for that, we will send you a new license key.

**Reply by diefallen** [reply]
> Hi, Sorry for that, we will send you a new license key.

received and safely activate it. again thanks chief!

**Comment by kxmxmxnn** [comment]
Hi I can't remember the purchase date
Can you give me my licence key, please?
kijun112@gmail.com

**Reply by jcfrog** [reply]
Hi,
I sent you an email.

**Reply by bagpiper3rd** [reply]
hey man i bought this for 55 dollars plus tax last year but i dont have the email address anymore. can you contact me to discuss this please?

**Reply by jcfrog** [reply]
hi,
please write to licenses@downloadhelper.net

**Comment by mouakka** [comment]
I bought the license yesterday on firefox when I noticed that I should buy a license to remove the watermark however when I had to install the companion app the extension became brand new and lost my premium tool now it is only downloading with big watermark (mouakkaanas@gmail.com)

**Reply by jcfrog** [reply]
Hi,
Sorry I did not find your license. Please write to licenses@downloadhelper.net for more details.

**Comment by LM610** [comment]
Hi, I just bought the mail license luigi.manzione@hotmail.com but it doesn't allow me to download videos. This window appears: "The license is for Edge but the extension version is Chrome". I bought it because I need to download several videos right away. How can I solve the problem?

**Reply by jcfrog** [reply]
Hi,
You may have installed the wrong extension. 
I recommend that you remove the extension and reinstall it from here: https://microsoftedge.microsoft.com/addons/detail/video-downloadhelper/jmkaglaafmhbcpleggkmaliipiilhldn

**Comment by hvelarde2024** [comment]
Hola, buenas tarde, compre una licencia de por vida y el donwloadhelper se activaba solo, nunca recibi la clave, me puede ayudar, mi correo es hvelarde426@gmail.com

**Reply by jcfrog** [reply]
Hi, 
Sorry for that, we will send you a new license key.

**Comment by shantiappslab** [comment]
Hola, Compré la licencia en CHrome hace años y no me acuerdo. Puedes enviarme la licencia a mi correo?

**Reply by jcfrog** [reply]
Hi,
please send us an email at licenses@downloadhelper.net

**Comment by michaelloya** [comment]
I purchase a chrome license and need it for Firefox. Please replace my Chrome license for a Firefox one?

teamloya@gmail.com

-ml

**Reply by jcfrog** [reply]
Hi,
I sent you an email.

**Comment by Ringorocks25** [comment]
What do we do if we bought a lifetime chrome license and now chrome forbids it to work? Can we switch the license to firefox?
Brian@brianervin.com

Thanks, Brian

**Reply by jcfrog** [reply]
Hi,
Sorry we can't swap licenses.
Chrome and Firefox premium are separate products with different prices and features.
If you purchased less than one month ago we can refund you so that you can purchase Firefox premium. You can write to licenses@downloadhelper.net

Note: Chrome never allowed downloads from Youtube, that's why Chrome premium is cheaper.

**Comment by mani268** [comment]


**Reply by jcfrog** [reply]
hi,
please send an email at licenses@downloadhelper.net

**Comment by kashishapp** [comment]


**Reply by jcfrog** [reply]
@mani268 @kashishapp are you the same user?

**Comment by MR-ROBOT990** [comment]
Hi please i need one please  here is my email : caneva.20pro@gmail.com

**Reply by jcfrog** [reply]
https://github.com/aclap-dev/video-downloadhelper/wiki#go-premium

**Comment by Delious83** [comment]
Hi, i bought mine via ChromeWebStore around 2026 , The Email was: dumu******@gmail.com
Can i get the licensekey aswell for manual activation? Thankyou

I use the product, it was fine, but now it says i need to restore licence, but there is no licence for me.  When i check the mail in your page it says i have firefox licence, but i am sure i bought chrome licence, it is written in the mail.

**Reply by jcfrog** [reply]
Please send an email to licenses@downloadhelper.net

**Comment by HAKumar-2024** [comment]
Hi I can't remember the purchase date
Can you give me my licence key, please?
kprologic@hotmail.com

**Reply by jcfrog** [reply]
Hi,
Did you try Restore purchase ? : https://help.downloadhelper.net/article/6-what-is-vdh#Register-your-premium-hb8sv
